### PR TITLE
Adapt harmonie processing to new harmonie version.

### PR DESCRIPTION
**Pull request by arjanverkerk**

### CODE REVIEW

File: raster_feeder/harmonie/config.py [hunk 1]
```diff
@@ -71,8 +76,9 @@
 PROJECTION = 'EPSG:4326'
 
 # remote file name strftime() format
-FORMAT = 'harm36_v1_ned_surface_%Y%m%d%H.tgz'
-PATTERN = r'harm36_v1_ned_surface_[0-9]{10}\.tgz'
+harm40_v1_p1_2019112018.tar
```
Comment (lexvand) [inline raster_feeder/harmonie/config.py]: Should be removed, right?

**Review by lexvand — CHANGES_REQUESTED** [review]
One line that isn't supposed to be there, I guess. Other than that it seems alright to me, for as far as I can tell.

File: raster_feeder/harmonie/config.py (hunk 1 above)
Comment (arjanverkerk) [reply raster_feeder/harmonie/config.py]: Very true. And now the tests run, too!

**Comment by arjanverkerk** [comment]
Care to approve or merge?

**Review by lexvand — APPROVED** [review]

**Pull request merged**